t3f/ops.py

```diff
@@ -790,32 +790,43 @@ def multiply(tt_left, right):
 
 
 def frobenius_norm_squared(tt, differentiable=False):
-  """Frobenius norm squared of a TensorTrain (sum of squares of all elements).
+  """Frobenius norm squared (sum of squares of all elements) of a TensorTrain
```

Comment:
The first sentence has to fit one line (80 symbols). It's treated as the short description.